src/components/new-safe/create/steps/ReviewStep/index.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { type ChainInfo } from '@safe-global/safe-gateway-typescript-sdk'
+
+import { render } from '@/tests/test-utils'
+import { NetworkFee } from '@/components/new-safe/create/steps/ReviewStep/index'
+import * as useWallet from '@/hooks/wallets/useWallet'
+import { type ConnectedWallet } from '@/services/onboard'
+import { ONBOARD_MPC_MODULE_LABEL } from '@/services/mpc/module'
+
+const mockChainInfo = {
+  chainId: '5',
+  l2: false,
+  nativeCurrency: {
+    symbol: 'ETH',
+  },
+} as ChainInfo
+
+describe('NetworkFee', () => {
+  it('displays the total fee if not social login', () => {
```

Comment:
Nit: I think usually we chose this form for our test expectations.

```suggestion
  it('should display the total fee if not social login', () => {
```